readthedocs/api/v2/views/integrations.py

```diff
@@ -74,14 +72,6 @@ class WebhookMixin:
     integration = None
     integration_type = None
     invalid_payload_msg = 'Payload not valid'
-    missing_secret_for_pr_events_msg = dedent(
-        """
-        This webhook doesn't have a secret configured.
-        For security reasons, webhooks without a secret can't process pull/merge request events.
-        For more information, read our blog post: https://blog.readthedocs.com/security-update-on-incoming-webhooks/.
-        """
-    ).strip()
```

Comment:
Are all the webhooks already migrated to those with a secret?

This message still seems useful to users that haven't migrated yet. If we remove it, how they will know they need to upgrade their integration?

Reply:
We show a similar message now https://github.com/readthedocs/readthedocs.org/blob/ff7a3b7cd393aa05e14116055e8d9d23a9c83438/readthedocs/api/v2/views/integrations.py#L75-L81

Reply:
Hrm, it seems that message is missing translation.